components/create-bounty-details.tsx

```diff
@@ -15,6 +20,36 @@ export default function CreateBountyDetails({
   review = false,
 }) {
   const { t } = useTranslation("bounty");
+  const [errorDescription, setErrorDescription] = useState<boolean>(false);
+  const [bodyLength, setBodyLength] = useState<number>(0);
+  const {
+    state: {
+      Settings
+    },
+  } = useAppState();
+
+  function handleChangeDescription (e: React.ChangeEvent<HTMLTextAreaElement>) { 
+    setBountyDescription(e.target.value)
+  }
+
+  useEffect(() => {
+    const strFiles = files?.map((file) =>
+        file.uploaded &&
+        `${file?.type?.split("/")[0] === "image" ? "!" : ""}[${file.name}](${
+          Settings?.urls?.ipfs
+        }/${file.hash}) \n\n`);
+
+    const body = `${bountyDescription}\n\n${strFiles
+          .toString()
+          .replace(",![", "![")
+          .replace(",[", "[")}`
+
+    body?.length && setBodyLength(body.length)
+
+    if(body.length > BODY_CHARACTERES_LIMIT && !errorDescription) setErrorDescription(true)
```

Comment:
this can be done directly on the html as we should know the `body.length` and the `filesLenght`, summing the two would be enough to show an error -- there shouldn't be a need for a `setErrorDescription`